app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt

```diff
@@ -398,6 +413,7 @@ private fun SelfUserProfileScreenPreview() {
             ),
             statusDialogData = null
         ),
+        isUserInCall = { false }
```

Comment:
Small thingy, could we extract this to a function and pass it as reference ? So we don´t have recompositions (I hear the voice of Mateuz again 😅 )

Reply:
it's extracted to function, but this is the preview only 😅